privacy/signal.md

```diff
@@ -26,7 +26,7 @@ What is [Signal]?
 How to install Signal in Qubes
 ------------------------------
 
-If you're a Signal user on Android, you can now have Signal inside Qubes.
+Oepn Whisper Systems develops a desktop version of Signal with the same text messaging functionality as the iPhone and Android apps. Signal for desktops is a Chrome app, which means it requires Google Chrome or its open source alternative Chromium to function. In order to install it on Qubes, you'll need to complete these steps:
```

Comment:
Minor point: It doesn't actually have the same functionality (e.g., no voice or video calling).

Reply:
Ok, I guess I interpreted "text messaging functionality" in a broad sense. For example, someone might say, "The Foo messaging app has text messaging functionality that includes emojis, voice chat, and soon video chat." This seems to be a way people speak and write these days. I see now that you mean strictly text messaging only.